src/main.rs

```diff
@@ -162,14 +176,41 @@ async fn main() {
     let default_keypair = args.keypair.unwrap_or(cli_config.keypair_path.clone());
     let fee_payer_filepath = args.fee_payer.unwrap_or(default_keypair.clone());
     let rpc_client = RpcClient::new_with_commitment(cluster, CommitmentConfig::confirmed());
-
+    let jito_client =
+        RpcClient::new("https://mainnet.block-engine.jito.wtf/api/v1/transactions".to_string());
+
+    let tip = Arc::new(RwLock::new(0_u64));
+    let tip_clone = Arc::clone(&tip);
+
+    if args.jito {
+        let url = "ws://bundles-api-rest.jito.wtf/api/v1/bundles/tip_stream";
+        let (ws_stream, _) = connect_async(url).await.unwrap();
```

Comment:
should probably throw this into the loop w/ back off on error so you get reconnected if we restart the server